### PR TITLE
Added hw 21 solution

### PR DESCRIPTION
**Pull request by programmer28**
1. Connected to MySQL server
2. Created AbstractDao class
3. Changed CRUD methods of Item`s model

### CODE REVIEW

File: src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+import mate.academy.internetshop.dao.ItemDao;
+import mate.academy.internetshop.lib.Dao;
+import mate.academy.internetshop.model.Item;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+@Dao
+public class ItemDaoJdbcImpl extends AbstractDao<Item> implements ItemDao {
+    private static Logger logger = LogManager.getLogger(ItemDaoJdbcImpl.class);
+    private static String DB_NAME = "test";
```
Comment (Tarabarnik) [inline src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java]:
```suggestion
    private static final String DB_NAME = "test";
```

File: src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+    public ItemDaoJdbcImpl(Connection connection) {
+        super(connection);
+    }
+
+    @Override
+    public Item create(Item item) {
+        String query = "INSERT INTO " + DB_NAME + ".items(name, price) VALUES('"
```
Comment (Tarabarnik) [inline src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java]: use String.format please

File: src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java (hunk 1 above)
Comment (programmer28) [reply src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java]: I`ve changed it

File: src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java (hunk 2 above)
Comment (programmer28) [reply src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java]: I`ve changed it

**Pull request merged**